### ISSUE TITLE
Update theme/template selection in Federalist

### ISSUE THREAD

**Issue by eddietejeda** [18F/federalist#2115]
#### User Story

As a user to Federalist who wants to launch a new site, I want to see all the theme/template options (including federalist-uswds-gatsby and federalist-uswds-jekyll) available on the admin interface.

#### Background (Optional)

The themes listed on Federalist are out of data and not yet incorporate the latest USWDS themes.

#### Acceptance Criteria

- [x] federalist-gatsby and federalist-uswds themes/templates are visible and able to be selected on Federalist
- [x] Determine and add language to frame the different templates

**Comment by davemcorwin** [comment]
On staging
USWDS v2.0 template site and repo created sucessfully.
Gatsby template repo was created but build failed with:
```
Configuration file: /work/site_repo/_config.yml
            Source: /work/site_repo
       Destination: /work/site_repo/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
jekyll 3.8.5 | Error:  No such file or directory @ rb_file_s_stat - /work/site_repo/public/0-d94c4e768dcdbc29a081.js
```

**Comment by eddietejeda** [comment]
I cleaned up the language a bit more. Can we work on this branch instead? https://github.com/18F/federalist/pull/2164

**Comment by eddietejeda** [comment]
This was resolved in #2164
